app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = list()
+
+    def __init__(
+        self,
+        name: str,
+        health: int = 100,
+        hidden: bool = False
+    ) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        me = f"Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}"
```

Comment:
why not used return f"Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}" ??